akka-javasdk/src/main/scala/akka/javasdk/impl/Validations.scala

```diff
@@ -4,63 +4,14 @@
 
 package akka.javasdk.impl
 
-import java.lang.reflect.AnnotatedElement
-import java.lang.reflect.Method
-import java.lang.reflect.ParameterizedType
-
-import scala.annotation.nowarn
-import scala.reflect.ClassTag
-
 import akka.annotation.InternalApi
-import akka.javasdk.agent.Agent
-import akka.javasdk.annotations.AgentDescription
-import akka.javasdk.annotations.AgentRole
-import akka.javasdk.annotations.Component
-import akka.javasdk.annotations.ComponentId
-import akka.javasdk.annotations.Consume.FromKeyValueEntity
-import akka.javasdk.annotations.Consume.FromWorkflow
-import akka.javasdk.annotations.FunctionTool
-import akka.javasdk.annotations.Produce.ServiceStream
-import akka.javasdk.annotations.Query
-import akka.javasdk.annotations.Table
-import akka.javasdk.consumer.Consumer
-import akka.javasdk.eventsourcedentity.EventSourcedEntity
-import akka.javasdk.impl.ComponentDescriptorFactory.eventSourcedEntitySubscription
-import akka.javasdk.impl.ComponentDescriptorFactory.findKVEClass
-import akka.javasdk.impl.ComponentDescriptorFactory.findWorkflowClass
-import akka.javasdk.impl.ComponentDescriptorFactory.hasAcl
-import akka.javasdk.impl.ComponentDescriptorFactory.hasAgentEffectOutput
-import akka.javasdk.impl.ComponentDescriptorFactory.hasConsumerOutput
-import akka.javasdk.impl.ComponentDescriptorFactory.hasESEffectOutput
-import akka.javasdk.impl.ComponentDescriptorFactory.hasEventSourcedEntitySubscription
-import akka.javasdk.impl.ComponentDescriptorFactory.hasHandleDeletes
-import akka.javasdk.impl.ComponentDescriptorFactory.hasKVEEffectOutput
-import akka.javasdk.impl.ComponentDescriptorFactory.hasKeyValueEntitySubscription
-import akka.javasdk.impl.ComponentDescriptorFactory.hasQueryEffectOutput
-import akka.javasdk.impl.ComponentDescriptorFactory.hasStreamSubscription
-import akka.javasdk.impl.ComponentDescriptorFactory.hasSubscription
-import akka.javasdk.impl.ComponentDescriptorFactory.hasTimedActionEffectOutput
-import akka.javasdk.impl.ComponentDescriptorFactory.hasTopicPublication
-import akka.javasdk.impl.ComponentDescriptorFactory.hasTopicSubscription
-import akka.javasdk.impl.ComponentDescriptorFactory.hasUpdateEffectOutput
-import akka.javasdk.impl.ComponentDescriptorFactory.hasWorkflowEffectOutput
-import akka.javasdk.impl.ComponentDescriptorFactory.hasWorkflowSubscription
-import akka.javasdk.impl.reflection.Reflect
-import akka.javasdk.impl.reflection.Reflect.Syntax._
-import akka.javasdk.keyvalueentity.KeyValueEntity
-import akka.javasdk.timedaction.TimedAction
-import akka.javasdk.view.TableUpdater
-import akka.javasdk.view.View
-import akka.javasdk.workflow.Workflow
 
 /**
  * INTERNAL API
  */
 @InternalApi
 private[javasdk] object Validations {
 
-  import Reflect.methodOrdering
-
   object Validation {
```

Comment:
Runtime validation are almost gone except for some for HttpEndpoint. I prefer to tackle this on follow-up PR. This PR is huuuge already.